### PR TITLE
issue #54: Wrapped OpenTimer into timing namespace.

### PR DESCRIPTION
**Pull request by csguth**
- Added libgoogle-glog as dependency on travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +37,6 @@ before_script:
   - cmake -DCMAKE_CXX_COMPILER=$COMPILER -DCMAKE_BUILD_TYPE=Release ..
 
 script:
-  - make -j
```
Comment (TiagoAFontana) [inline .travis.yml:36]: why not "make -j"?

Comment (renannetto) [inline .travis.yml:36]: what is this libgoogle-glog-dev package?

**Comment by csguth** [comment]
@TiagoAFontana I'm gonna revert the make thing.

@renannetto:
Glog is a logging library, used by OpenTimer. Since I got rid of autotools (it's really hard to integrate autotools and cmake perfectly) I found easier just to link OpenTimer with the system's libglog. I also added the find command on OpenTimer's cmake. Hence, to allow the build on Travis, I had to add the dependency on .travis.yml file.

**Review by renannetto — APPROVED** [review]

**Comment by renannetto** [comment]
There are merge conflicts. I guess you have to merge it with dev before.

**Pull request merged**